rbd/pool_nautilus.go

```diff
@@ -122,3 +122,61 @@ func PoolStatsCreate() *PoolStats {
 func (poolStats *PoolStats) Destroy() {
 	C.rbd_pool_stats_destroy(poolStats.stats)
 }
+
+// PoolStatOptions represents a group of configurable pool stat options.
+type PoolStatOptions C.rbd_pool_stat_option_t
```

Comment:
Is this a single option? Call it `PoolStatOption`.

Reply:
Fixed